backend/capellacollab/__main__.py

```diff
@@ -31,7 +31,6 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 logging.getLogger("uvicorn.access").addFilter(HealthcheckFilter())
 log.addFilter(HealthcheckFilter())
-logging.getLogger(__name__).setLevel(config["logging"]["level"])
```

Comment:
This line is still needed if the Collaboration Manager is started from a context that is not alembic.